doc/CONTRIBUTING.md

```diff
@@ -7,6 +7,10 @@ Contributions are welcome! Please consider the following guidelines.
 - Run `make format` to automatically format your code.
 - Any changes to code style should be submitted as pull requests to https://github.com/cryptiumlabs/ormolu
 
+## Documentation
+
+Add a brief description of any module you create before the module and after pragmas. Start each line of the description with `--`. The first line of the description should be `--|`. E.g., see [this](https://github.com/cryptiumlabs/juvix/blob/15ca9e5e602d24cf09fe87fc059e3e0ee78ad6db/src/Juvix/Encoding/Encoding.hs#L3).
+
```

Comment:
Seems like a good start, I can probably add details to this soonish.

Reply:
OK, or you can add 'add suggestions' and I can apply them in this PR too.